SimplenoteUITests/Settings.swift

```diff
@@ -0,0 +1,48 @@
+import XCTest
+
+let screenName = "Settings"
+
+enum SettingsScreen: String {
+
+    case doneButton = "Done"
+    case condensedModeSwitch = "Condensed Note List"
+
+    var element: XCUIElement {
+        switch self {
+        case .doneButton:
+            return app.navigationBars[screenName].buttons[self.rawValue]
+        case .condensedModeSwitch:
+            return app.tables.cells.containing(.staticText, identifier: self.rawValue).firstMatch
```

Comment:
This is to return the parent of the `Condensed Note List` static text. The parent cell contains a useful 0 or 1 value, while the static text has no properties allowing to tell if the switch is on or off.